pkgs/applications/virtualization/lima/default.nix

```diff
@@ -14,16 +14,16 @@
 
 buildGoModule rec {
   pname = "lima";
-  version = "1.0.7";
+  version = "1.1.0";
```

Comment:
In v1.1, `lima-additional-guestagents` (`make additional-guestagents`) should be packaged separately from `lima` (`make native`).



- https://github.com/lima-vm/lima/issues/3563

Homebrew:
- https://github.com/Homebrew/homebrew-core/pull/224206

Reply:
If I get it right, the update (and thus PR) is blocked on splitting the additional guestagents?

Reply:
I'm addressing this updates and splitting in https://github.com/NixOS/nixpkgs/pull/415093

Initially, I considered that adding `withAdditionalGuestAgent` inputs or the `limaMinimal` package would be better for backward compatibility. However, I've concluded that simply splitting it, similar to the upstream approach, is good. Please let me know if my understanding is incorrect.